examples/api/amp-access/amp-access.js

```diff
@@ -38,10 +38,10 @@ const POWER_USERS = {
 };
 const EXPIRATION_DATE = 24*60*60*1000; // 1 day in ms
```

Comment:
please rename this file from amp-access.js to index.js.

Reply:
Thanks for pointing that out. PTAL.